heroku.md

```diff
@@ -6,9 +6,7 @@ Here's how to deploy your implementation of, say, [C$50 Finance](https://cs50.ha
 
 1. Watch [Brian's seminar](https://youtu.be/MJUJ4wbFm_A) to learn about `git` and GitHub, if not already familiar.
 
-1. Sign up for free, private GitHub repositories at <https://education.github.com/discount_requests/new>, if you haven't already.
-
-1. Create a new **private** repository at <https://github.com/new> (called, e.g., `finance`).
+1. Sign into your GitHub account and create a new **private** repository at <https://github.com/new> (called, e.g., `finance`).
```

Comment:
I think we still need the other step?

> 1. Create a new **private** repository at <https://github.com/new> (called, e.g., `finance`).

Reply:
I didn't remove that.  I just added "Sign into your GitHub account and  create ... "

Reply:
Seems to have been deleted! 

<img width="736" alt="Screen Shot 2021-01-04 at 11 43 41 PM" src="https://user-images.githubusercontent.com/788678/103607355-b15c6880-4ee6-11eb-89b1-76fef2c1f055.png">

Reply:
It's still there. :)